scripts/next-data/generatePreBuildFiles.mjs

```diff
@@ -1,8 +1,13 @@
 // @TODO: This is a temporary hack until we migrate to the `nodejs/nodejs.dev` codebase
-import { writeFile } from 'fs/promises';
+import { writeFile, readFile} from 'fs/promises';
 import { join } from 'path';
 import { Feed } from 'feed';
-
+import { evaluate } from '@mdx-js/mdx'
+import { createElement } from 'react'
+import { renderToString } from 'react-dom/server'
+import remarkGfm from 'remark-gfm'
```

Comment:
Do we need remark and remark-gfm here? I thought mdx would already include remark built-ins.